src/litDataset.ts

```diff
@@ -408,6 +409,38 @@ export async function saveLitDatasetInContainer(
   return resourceWithResolvedIris;
 }
 
+/**
+ * Save the ACL for a Resource.
+ *
+ * @param resource The Resource to which the given ACL applies.
+ * @param resourceAcl An [[unstable_AclDataset]] whose ACL Rules will apply to `resource`.
+ * @param options Optional parameter `options.fetch`: An alternative `fetch` function to make the HTTP request, compatible with the browser-native [fetch API](https://developer.mozilla.org/docs/Web/API/WindowOrWorkerGlobalScope/fetch#parameters).
+ */
+export async function unstable_saveAclFor(
+  resource: WithResourceInfo & {
```

Comment:
This type is a little hard to follow - with unions and excludes, at what point does it just make sense to create a new type (even if this is the implementation underneath)? Could be my unfamiliarity with Typescript, but it _feels_ like a smell if your type definition is 7 lines long.

Reply:
Yep, that's fair - in fact, I think I did want to create a separate type alias for that after I finished it, but forgot. I'll do it in a separate PR, since, you know... It involves new names :P

Reply:
https://github.com/inrupt/lit-pod/pull/206